js/components/ChooseAvatar/styles.js

```diff
@@ -51,20 +56,24 @@ const styles = StyleSheet.create({
     height: height / 9
   },
   editAvatar: {
+    marginLeft: 20,
+    marginRight: 20,
+    marginTop: 15,
+    marginBottom: 15,
```

Comment:
use flexbox instead of margin in all directions.

Reply:
the margins are to push the border around the avatar. If I use flex I have to give a height and width to a TouchableOpacity. This is the best way to reflect the design.

Reply:
It's ok to give height and width to Touchable Opacity. Make the changes as requested.